coq.opam

```diff
@@ -20,11 +20,9 @@ homepage: "https://coq.inria.fr/"
 doc: "https://coq.github.io/doc/"
 bug-reports: "https://github.com/coq/coq/issues"
 depends: [
-  "ocaml" {>= "4.05.0"}
-  "dune" {>= "2.5.0"}
-  "ocamlfind" {>= "1.8.1"}
-  "zarith" {>= "1.10"}
-  "ounit2" {with-test}
+  "dune" {>= "2.5"}
+  "coq-core" {= version}
+  "coq-stdlib" {= version}
 ]
 build: [
```

Comment:
Shouldn't you remove all build steps to make this a proper virtual package? And remove `dune` from the list of dependencies?

Reply:
Umm, good point; only point here would be to have to install the `dune-package` file, will ask.

Reply:
I am gonna leave it like this, as this could be useful for example to have the coq package run the test suite, but indeed we may want to further tweak; but indeed the dune subst, etc... is required to have proper pinning of Opam packages, so I think that due to this bug we should bump to 2.8 which while not ideal, it is convenient IMHO.